docs/modules/optim.rst

```diff
@@ -26,9 +29,9 @@
 
 .. raw:: html
 
-  CollaborativeOptimizer is a legacy version of hivemind.Optimizer. **For new projects, please use hivemind.Optimizer.**
```

Comment:
I suggest to move it right under the class definition, since people often only read the class description (but not the section header).

Reply:
[done]